config/deploy/dev.rb

```diff
@@ -0,0 +1 @@
+server 'upaya-dev.18f.gov', roles: %w(web app db)
```

Comment:
I wonder if it's because it's set to `:ubuntu` instead of `'ubuntu'` in the global config.